### PR TITLE
add cumprod and cumsum funcs

### PR DESCRIPTION
**Pull request by KsanaKozlova**

### CODE REVIEW

**Review by densmirn — COMMENTED** [review]
Maybe unskip some related tests in `tests/skipped_tests.py`/`tests/skipped_tests_gpu.py`?

**Comment by samir-nasibli** [comment]
Please, provide some tests for check.

File: dpnp/dpnp_iface_mathematical.py [hunk 1]
```diff
@@ -253,6 +255,78 @@ def copysign(x1, x2, **kwargs):
...
+    if not use_origin_backend(x) and not kwargs:
+        if not isinstance(x, dparray):
+            pass
+        else:
+            return dpnp_cumprod(x)
+
+    return call_origin(numpy.cumprod, x, **kwargs)
+
+
+def cumsum(x, **kwargs):
```
Comment (shssf) [inline dpnp/dpnp_iface_mathematical.py:294]: please don't use single letter variable names. May be `x1` is a bit better than just an `x`

File: dpnp/backend_mathematical.pyx [hunk 2]
```diff
@@ -108,6 +110,54 @@ cpdef dparray dpnp_copysign(dparray x1, dparray x2):
     return call_fptr_2in_1out(DPNP_FN_COPYSIGN, x1, x2, x1.shape)
 
 
+cpdef dparray dpnp_cumprod(dparray x1):
+
+    x1_dtype = x1.dtype
```
Comment (densmirn) [inline dpnp/backend_mathematical.pyx:115]: I'm not sure variable `x1_dtype` is needed. `x1.dtype` may be used directly. The same is in `dpnp_cumsum`.

**Review by densmirn — APPROVED** [review]

**Review by densmirn — APPROVED** [review]

**Pull request merged**